lib/runner/androidEmulator.js

```diff
@@ -0,0 +1,80 @@
+const untildify = require('untildify');
+const {Logger} = require('../utils');
+const path = require('path');
+
+module.exports = class MobileServer {
+
+  constructor (AVD) {
+    const {getPlatformName, getBinaryLocation} = require('@nightwatch/mobile-helper');
+
+    this.sdkRoot = this.getSdkRootFromEnv(),
+    this.platform = getPlatformName();
+    this.binaryLocation = getBinaryLocation;
+    this.avd = AVD || 'nightwatch-android-11';
+    this.emulatorId = '';
+    this.emulatorAlreadyRunning = false;
+  }
+
+  getSdkRootFromEnv() {
+    const androidHome = process.env.ANDROID_HOME;
+  
+    if (androidHome) {
+      return untildify(androidHome);
```

Comment:
You should return `path.resolve(untildify(androidHome))` instead. Maybe bring back some variable names so it is more readable?